gemrb/plugins/GUIScript/GUIScript.cpp

```diff
@@ -1638,6 +1638,48 @@ static PyObject* GemRB_Control_SetText(PyObject* self, PyObject* args)
 	Py_RETURN_NONE;
 }
 
+PyDoc_STRVAR( GemRB_Window_SetKeyPressEvent__doc,
```

Comment:
this is at the very least going to need a rewrite since we don't have a "window index" anymore. nor does `core` have APIs for getting at the windows.

Can't this be handled by `Window::RegisterHotKeyCallback`? We have a Python variant for `EventCallback`